proto/api/v2/compliance_scan_configuration_service.proto

```diff
@@ -188,12 +188,17 @@ service ComplianceScanConfigurationService {
 
   // GetReportHistory returns a list of snapshots (scan executions) from a given scan configuration.
   rpc GetReportHistory(ComplianceReportHistoryRequest) returns (ComplianceReportHistoryResponse) {
-    option (google.api.http) = {get: "/v2/compliance/scan/configuration/{id}/reports/history"};
```

Comment:
Did this endpoint exist before now?  If so, I'm fine changing the endpoint because we are in tech preview, but we should note the change in the CHANGELOG.

Reply:
This endpoint is new so I don't think we need add anything in the CHANGELOG. That reminded me though that we need to add a line for the change in the run report endpoint because we changed the method from put to post. I'll open a PR for that.

Reply:
Changelog PR #13114